Makefile.am

```diff
@@ -45,6 +45,15 @@ src_libfabric_la_SOURCES = \
 	src/log.c \
 	$(common_srcs)
 
+if MACOS
+src_libfabric_la_SOURCES += src/osx/osd.c
+src_libfabric_la_SOURCES += include/osx/osd.h
+else
+if LINUX
```

Comment:
You are correct -- I stand corrected!

FWIW, I'd de-tangle the conditionals -- the MACOS and LINUX conditionals don't need to be nested.